examples/app/headless_renderer.rs

```diff
@@ -0,0 +1,444 @@
+//! This example illustrates how to make headless renderer
+
+mod frame_capture {
```

Comment:
I'm not a big fan of starting the example with the sub-module definition. I'd prefer if the classic bevy main setup was up there like other examples.